templates/rke2/overlay/opt/corral/rke2/init-cluster.sh

```diff
@@ -11,6 +13,19 @@ tls-san:
   - ${CORRAL_api_host}
 EOF
 
+else
+
+mkdir -p /etc/rancher/rke2
+cat > /etc/rancher/rke2/config.yaml <<- EOF
+write-kubeconfig-mode: 644
+cni: ${CORRAL_cni}
+system-default-registry: ${CORRAL_registry_fqdn}
+tls-san:
+  - ${CORRAL_api_host}
+EOF
+
+fi
```

Comment:
This is from Izaac's work. Should probably be addressed his PR, and I can rebase to receive the changes.

Reply:
I tried auto formatters, bash linters, manually checking for special hidden characters. But if I don't keep this the way it is I end up with the rke2 service not coming up:

```bash
DEBU[0465] [izaac-9e5f51742ab5-cp-1]: Job for rke2-server.service failed because the control process exited with error code.
DEBU[0465] [izaac-9e5f51742ab5-cp-1]: See "systemctl status rke2-server.service" and "journalctl -xe" for details.
```

Any other suggestions?

Reply:
Disregard, I know what the error is now.